pkg/operators/filter/filter.go

```diff
@@ -81,11 +83,22 @@ func (f *filterOperator) InstanceParams() api.Params {
 	}}
 }
 
+func keyForDataSource(datasources map[string]datasource.DataSource, ds datasource.DataSource) (string, string) {
+	key := ParamFilterExpr
+	alias := "E"
+	if len(datasources) != 1 {
+		key += fmt.Sprintf("-%s", ds.Name())
+		alias = ""
+	}
+	return key, alias
+}
```

Comment:
This is different from the usual syntax (which would be `--filter-expr ds1:filter-for-dn1,ds2:filter-for-ds2`). I don't think what we have now is optimal (and `--filter` seems to be handled slightly different as well) , but maybe we should still align it until we have a better solution?